lib/router.coffee

```diff
@@ -182,7 +182,7 @@ run = (defaults = {}) ->
 
     twitter = _.extend({}, defaults.twitter, seo.twitter)
     og = _.extend({}, defaults.og, seo.og)
-    meta = _.extend({}, defaults.meta, seo.meta)
+    meta = _.extend({}, call(seo.meta or defaults.meta))
```

Comment:
Will this behave in the same way as `meta = _.extend({}, defaults.meta, seo.meta)`? Looks like `seo.meta` will override everything from `defaults.meta` if it's present – not extending it?